### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by Hotanya**
## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/es-hangul/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

You can [this](https://github.com/projectdiscovery/nuclei/security) as a guideline for the contents of the SECURITY.md file.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Security Policy
```
Comment (okinawaa) [inline SECURITY.md:1]: how about this?

# Security Policy

## Reporting a Vulnerability

If you discover any security vulnerabilities in the `es-hangul` package, please report them immediately. We take security seriously and will address all legitimate reports in a timely manner.

### How to Report

To report a vulnerability, please email us at **[chanhyuk.park@toss.im]**. Provide as much detail as possible about the vulnerability, including:

- The nature of the vulnerability
- Steps to reproduce the issue
- Any potential risks or impacts on users
- Your contact information for further clarification

### Response Process

1. We will acknowledge receipt of your report within **48 hours** and begin investigating the issue.
2. After validating the report, we will work to fix the vulnerability and release an update as soon as possible.
3. You will be informed of the resolution once the fix is deployed and may be credited in the release notes if desired.
4. Security patches will be communicated through GitHub releases and other relevant channels.

## Security Best Practices

We recommend following these best practices to help maintain the security of your application when using `es-hangul`:

- Always use the latest version of `es-hangul`.
- Regularly update your dependencies to include the latest security fixes.
- Review and monitor your own usage for potential security issues.

If you have any questions or need further information, please don't hesitate to contact us.

**Review by okinawaa — CHANGES_REQUESTED** [review]
We don't have supported versions 
We have to do that?

File: SECURITY.md (hunk 1 above)
Comment (Hotanya) [reply SECURITY.md:1]: Yup that's great, thanks!

**Comment by Hotanya** [comment]
> We don't have supported versions 
> 
> We have to do that?

You could change the wording in the file to say something about it not being supported? But at least having the security.md file allows vulnerabilities to be reported through GitHub.

File: SECURITY.md (hunk 1 above)
Comment (okinawaa) [reply SECURITY.md:1]: @Hotanya can you reflect this comment in this PR?

Comment (Hotanya) [reply SECURITY.md:1]: I have updated the PR accordingly.

**Review by okinawaa — APPROVED** [review]
very thanks!!!

**Pull request merged**